### PR TITLE
Remove "base64-encoded" from source provider instructions

### PR DESCRIPTION
**Pull request by RichardHoch**
Bug fix.

MTV 2.3
Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2067054

Remove "base64-encodes" from step 2 of the procedure for adding a source provider.

Preview: https://deploy-preview-299--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#migrating-virtual-machines-cli_forklift

### CODE REVIEW

File: documentation/modules/migrating-virtual-machines-cli.adoc
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:60]: Please change it to: "Specify the user password"

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Review by mguetta1 — APPROVED** [review]
Looks good to me

**Comment by RichardHoch** [comment]
@stoobie Ready for your review. This is for both versions 2.2 and 2.3.

**Review by stoobie — APPROVED** [review]
LGTM. We'll need to cherry pick this to the 2.3 branch.

Before merging, let's wait for @mguetta1 to approve your latest changes.

File: documentation/modules/migrating-virtual-machines-cli.adoc
Comment (stoobie) [inline documentation/modules/migrating-virtual-machines-cli.adoc:61]: @mguetta1 Do you approve?

File: documentation/modules/migrating-virtual-machines-cli.adoc
Comment (stoobie) [inline documentation/modules/migrating-virtual-machines-cli.adoc:83]: @mguetta1 Do you approve?

**Review by mguetta1 — COMMENTED** [review]
@RichardHoch @stoobie There are more places where "www.example.com" is mentioned, do you want to fix them in this PR or in another one?

**Comment by RichardHoch** [comment]
@stoobie She asked me to make the changes offline. We're good. Another BZ deals with "fqdn" globally (in the repo, not all repos).

**Comment by RichardHoch** [comment]
> @RichardHoch @stoobie There are more places where "www.example.com" is mentioned, do you want to fix them in this PR or in another one?

Taking care of all in MTV repo in this BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2067152

**Review by stoobie — COMMENTED** [review]
@RichardHoch One more comment.

File: documentation/modules/migrating-virtual-machines-cli.adoc
Comment (RichardHoch) [reply documentation/modules/migrating-virtual-machines-cli.adoc:61]: @istein1 @mguetta1 
1. Should line 60 read "Specify the user password" or "Specify the admin password"?
2. Does the answer to question apply to both MTV 2.2 and MTV 2.3?

Comment (mguetta1) [reply documentation/modules/migrating-virtual-machines-cli.adoc:61]: @RichardHoch I think it fits for both 2.2 and 2.3
The thing is that regarding **VMware only** - in 2.2 the requirement is to use admin user (mentioned in (1) in 2.2 doc)
and for 2.3 it can be any user (with required permissions for MTV migration - mentioned in #282 PR)
In my opinion, "Specify the user password" is OK for both versions, as long we mention in (1) in 2.2 doc: "Specify the vCenter **admin** user or..."

**Comment by RichardHoch** [comment]
@stoobie This is ready to be merged but it is for versions 2.2 and 2.3.

File: documentation/modules/migrating-virtual-machines-cli.adoc
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:61]: @RichardHoch please remove the "fqdn" change, it is handled in other PR & BZ

File: documentation/modules/migrating-virtual-machines-cli.adoc
Comment (mguetta1) [reply documentation/modules/migrating-virtual-machines-cli.adoc:83]: @RichardHoch please remove the "fqdn" change, it is handled in other PR & BZ

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Comment by mguetta1** [comment]
Hi @RichardHoch this PR is for MTV 2.3, we have this PR for 2.2: #307

**Comment by RichardHoch** [comment]
@mguetta1 Please check this -- it's for version 2.3 only.

**Review by mguetta1 — APPROVED** [review]
Looks good to me

**Comment by RichardHoch** [comment]
This PR was closed because of a problem we had with git. I duplicated the changes here in a second PRhttps://github.com/konveyor/forklift-documentation/pull/315.

That PR was successfully merged to the 2.3 branch.

**Pull request closed without merging**
